cluster/src/main/java/com/hartwig/pipeline/tertiary/orange/Orange.java

```diff
@@ -0,0 +1,246 @@
+        this.sageGermlineGeneCoverageTsv = new InputDownload(sageGermlineOutput.germlineGeneCoverageTsv());
+        this.sageSomaticRefSampleBqrPlot = new InputDownload(sageSomaticOutput.somaticRefSampleBqrPlot());
+        this.sageSomaticTumorSampleBqrPlot = new InputDownload(sageSomaticOutput.somaticTumorSampleBqrPlot());
+        this.linxOutputDir = new InputDownload(linxOutput.linxOutputLocations().outputDirectory());
+        this.linxFusionTsv = new InputDownload(linxOutput.linxOutputLocations().fusions());
+        this.linxBreakEndTsv = new InputDownload(linxOutput.linxOutputLocations().breakends());
+        this.linxDriverCatalogTsv = new InputDownload(linxOutput.linxOutputLocations().driverCatalog());
+        this.linxDriverTsv = new InputDownload(linxOutput.linxOutputLocations().drivers());
+        this.chordPredictionTxt = new InputDownload(chordOutput.predictions());
+        this.cuppaConclusionTxt = new InputDownload(cuppaOutput.conclusionTxt());
+        this.cuppaResultCsv = new InputDownload(cuppaOutput.resultCsv());
+        this.cuppaSummaryPlot = new InputDownload(cuppaOutput.chartPng());
+        this.cuppaFeaturePlot = new InputDownload(cuppaOutput.featurePlot());
+        this.peachGenotypeTsv = new InputDownload(peachOutput.genotypeTsv());
+        this.protectEvidenceTsv = new InputDownload(protectOutput.evidenceTsv());
+        this.annotatedVirusTsv = new InputDownload(virusOutput.annotatedVirusFile());
+    }
+
+    @Override
+    public List<BashCommand> inputs() {
+        return List.of(purpleSomaticVcf,
+                refMetrics,
+                tumMetrics,
+                refFlagstat,
+                tumFlagstat,
+                sageGermlineGeneCoverageTsv,
+                sageSomaticRefSampleBqrPlot,
+                sageSomaticTumorSampleBqrPlot,
+                purpleOutputDir,
+                purpleGermlineVcf,
+                purplePurityTsv,
+                purpleQCFile,
+                purpleGeneCopyNumberTsv,
+                purpleSomaticDriverCatalog,
+                purpleGermlineDriverCatalog,
+                linxOutputDir,
+                linxFusionTsv,
+                linxBreakEndTsv,
+                linxDriverCatalogTsv,
+                linxDriverTsv,
+                cuppaConclusionTxt,
+                cuppaResultCsv,
+                cuppaSummaryPlot,
+                cuppaFeaturePlot,
+                chordPredictionTxt,
+                protectEvidenceTsv,
+                annotatedVirusTsv,
+                peachGenotypeTsv
+        );
+    }
+
+    @Override
+    public String namespace() {
+        return NAMESPACE;
+    }
+
+    @Override
+    public List<BashCommand> commands(final SomaticRunMetadata metadata) {
+        final String pipelineVersionFilePath = VmDirectories.OUTPUT + "/orange_pipeline.version.txt";
+
+        final String pipelineVersion = Versions.pipelineMajorMinorVersion();
+
```

Comment:
Nit: the blank lines actually hinder readability for me, maybe I'm the only one.